browser/ui/tabs/brave_tab_color_mixer.cc

```diff
@@ -98,21 +96,19 @@ void AddBraveTabDarkThemeColorMixer(ui::ColorProvider* provider,
                                     const ui::ColorProviderKey& key) {
   ui::ColorMixer& mixer = provider->AddMixer();
 
-  static const auto kDefaultColorMap =
-      base::MakeFixedFlatMapNonConsteval<BraveColorIds, SkColor>({
-          {kColorBraveVerticalTabActiveBackground,
-           mixer.GetResultColor(kColorTabBackgroundActiveFrameActive)},
-          {kColorBraveVerticalTabInactiveBackground,
-           mixer.GetResultColor(kColorToolbar)},
-          {kColorBraveVerticalTabSeparator,
-           SkColorSetA(SK_ColorWHITE, 0.1 * 255)},
-          {kColorBraveVerticalTabNTBIconColor,
-           SkColorSetA(SK_ColorWHITE, 0.6 * 255)},
-          {kColorBraveVerticalTabNTBTextColor, SkColorSetRGB(0x8C, 0x90, 0xA1)},
-          {kColorBraveVerticalTabNTBShortcutTextColor,
-           SkColorSetRGB(0x68, 0x6D, 0x7D)},
-          {kColorBraveSplitViewTileBackground, SkColorSetRGB(0x0D, 0x12, 0x14)},
-      });
+  static const base::flat_map<BraveColorIds, SkColor> kDefaultColorMap = {
```

Comment:
`base::NoDestructor`

Reply:
Updated in [3dee1cb](https://github.com/brave/brave-core/pull/22936/commits/3dee1cbed6672b5a8c42a02423a53e46bc806d7f)